core/src/main/resources/rmapcore.properties

```diff
@@ -14,7 +14,9 @@ rmapcore.defaultStatusFilter=active
 #written to the database. SELECT_ORDER is significantly performance-wise slower but will eliminate skipped 
 #or repeated records where reads and writes are happening simultaneously. 
 rmapcore.defaultOrderBy=UNORDERED
-
+# If you specify an Admin Agent URI, currently this Agent will be able to tombstone or hard delete a DiSCO on behalf 
+# of another Agent. In general Agents can only edit their own DiSCOs - this is the exception.  
+rmapcore.adminAgentUri=[YourRMapHomePage]#Administrator
```

Comment:
I'd prefer to leave it with a bad uri so that a user doesn't accidentally create the Admin Agent from an incorrect property.  They would have to configure this if using the admin tool.